app/pages/summary/complete/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,292 @@
+import nock from 'nock';
+import { ClientFunction, Selector } from 'testcafe';
+import { extractInnerText } from 'buying-catalogue-library';
+import content from '../manifest.json';
+import { orderApiUrl } from '../../../../config';
+import { formatDate } from '../../../../helpers/common/dateFormatter';
+import { nockAndErrorCheck, setState, authTokenInSession } from '../../../../test-utils/uiTestHelper';
+
+const pageUrl = 'http://localhost:1234/order/organisation/order-1/summary';
+
+const mocks = () => {
+  nock(orderApiUrl)
+    .get('/api/v1/orders/order-1')
+    .reply(200, { description: 'some order description', status: 'complete' });
+};
+
+const pageSetup = async (setup = { withAuth: true, getRoute: true }) => {
+  if (setup.withAuth) {
+    await setState(ClientFunction)('fakeToken', authTokenInSession);
+  }
+  if (setup.getRoute) {
+    mocks();
+  }
+};
+
+const getLocation = ClientFunction(() => document.location.href);
+
+fixture('Order Summary Summary - general')
```

Comment:
can you update this please. `Order Summary for Complete order - general` perhaps? also can you update the incomplete version of this to something similar.